app/models/fae/navigation.rb

```diff
@@ -69,9 +72,28 @@ def item(text, options={})
       # prefer `subitems` over `sublinks` for new DSL but still need to support old structure in v1
       hash[:sublinks] = hash[:subitems] if hash[:subitems].present?
 
+      # authorize link with current user
+      unless can_view_path(hash[:nested_path])
+        hash[:path] = '#'
```

Comment:
So only the path is hidden from unauth'd users? I think this is unnecessary - I'd rather click and get an unauthorized / 404 page than to click and think the link is broken. Actually, I'd vote to remove the nav item entirely if the user isn't authorized.